oracledb-ballerina/tests/02_execute-basic-test.bal

```diff
@@ -0,0 +1,386 @@
+ // Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ //
+ // WSO2 Inc. licenses this file to you under the Apache License,
+ // Version 2.0 (the "License"); you may not use this file except
+ // in compliance with the License.
+ // You may obtain a copy of the License at
+ // http://www.apache.org/licenses/LICENSE-2.0
+ //
+ // Unless required by applicable law or agreed to in writing,
+ // software distributed under the License is distributed on an
+ // "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ // KIND, either express or implied. See the License for the
+ // specific language governing permissions and limitations
+ // under the License.
+
+ import ballerina/sql;
+ import ballerina/test;
+
+ @test:Config {
+     enable: true,
+     groups:["execute","execute-basic"]
+ }
+ function testCreateTable() {
+     Client oracledbClient = checkpanic new(user, password, host, port, database);
```

Comment:
Shall we avoid using `checkpanic` in testcases and returns the error. check all the testcases

Reply:
Fixed with 57294320259b2614da3d4192b8456f89431641e4